dotCMS/src/main/java/com/dotcms/rest/api/v1/user/UserResource.java

```diff
@@ -424,11 +424,14 @@ public final Response logoutAs(@Context final HttpServletRequest request) {
 	 * currently logged-in user is automatically removed from the result list.
 	 *
 	 * @return The list of users that can be impersonated.
+	 *
+	 * @deprecated use {@link com.dotcms.rest.api.v2.user.UserResource#loginAsData(HttpServletRequest, String, int, int)}
 	 */
 	@GET
 	@Path("/loginAsData")
 	@JSONP
 	@NoCache
+	@Deprecated
 	@Produces({ MediaType.APPLICATION_JSON, "application/javascript" })
 	public final Response loginAsData(@Context final HttpServletRequest request, @QueryParam("filter") String filter,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)